### PR TITLE
Spacy3 benchmarks

### PR DESCRIPTION
**Pull request by qeterme**
Hi,
I've added scripts to benchmark quickly HuSpaCy with Stanza and UDPipe. It outputs the time spent, memory usage and how many tokens has been processed / sec in a Markdown table format.

I also added benchmarking scripts for HuSpaCy v2 and emtsv but those are not included in the bash file.

### CODE REVIEW

**Comment by oroszgy** [comment]
Nice job!

**Review by oroszgy — COMMENTED** [review]
Nice job!

File: huspacy/cli/benchmarks/emtsv_benchmark.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    print(f'Maximum memory usage: {resource.getrusage(resource.RUSAGE_SELF).ru_maxrss / 1024:.2f} MiB')
+
+
+def runner(output_file, sentences):
+    for tokenList in tqdm(sentences):
+        r = requests.post('http://127.0.0.1:5000/tok/morph/pos/emmorph2ud/dep/conll', data={'text': tokenList.metadata["text"]})
+        print(f"# sent_id = {tokenList.metadata['sent_id']}",f"# text = {tokenList.metadata['text']}",r.text, sep="\n", file=output_file)
+
+
+def make_it_valid():
```
[the remark this answers is not in the document]
Comment (qeterme) [reply huspacy/cli/benchmarks/emtsv_benchmark.py:49]: Yes, there's an error with emtsv. The conll formatted outputs have multiple roots in them and the conll18_ud_eval.py script cannot evaulate it.

File: huspacy/cli/benchmarks/huspacyv3_benchmark.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    nlp.add_pipe("conll_formatter")
+
+    with Timer() as t:
+        print(nlp(input)._.conll_str)
+
+    print(f'Time spent: {t.elapsed:.2f} seconds')
+    print(f'Maximum memory usage: {resource.getrusage(resource.RUSAGE_SELF).ru_maxrss / 1024:.2f} MiB')
+
+
+def get_nlp(gpu: bool, ner: bool):
```
[the remark this answers is not in the document]
Comment (qeterme) [reply huspacy/cli/benchmarks/huspacyv3_benchmark.py]: Fixed it!

**Pull request merged**